.gitattributes

```diff
@@ -0,0 +1,10 @@
+* text=auto
```

Comment:
The files/folder mentioned in this file will not be distributed when people do `composer require benjaminfavre/oauth2-http-client`. This means it will be faster to download the package and less effect on the environment ;)